script/README.md

```diff
@@ -0,0 +1,8 @@
+## Deploying the Axelar Transceiver
+
+First create a `.env` file in the root directory and add the following 3 parameters corresponding to the deployments on target chain.
+- AXELAR_GATEWAY = 0x...
+- AXELAR_GAS_SERVICE = 0x...
+- NTT_MANAGER = 0x...
+
+Then run `forge script --chain {chain_name} srcipt DeployAxelarTransceiver.sol` to deploy the Axelar Transceiver.
```

Comment:
add a script to set chain id mapping as well

Reply:
done